### PR TITLE
Add CI action to run Notebooks

### LINKED ISSUE #467: Check tutorials and remove `helper_functions`
Apparently I missed these obvious mistakes back when #453 was merged.

- [ ] Check and remove `helper_functions` from tutorial 0.
- [ ] Check and remove `helper_functions` from tutorial 1.

### PR DESCRIPTION
**Pull request by Rodrigo-Tenorio**
Testing the correctness of a tutorial is quite an involved task, so for now I'll implement a basic action item to make sure the notebooks run properly without exceptions.

Closes #467

### CODE REVIEW

**Comment by Rodrigo-Tenorio** [comment]
> https://github.com/PyFstat/PyFstat/actions/runs/3135081203 shows "3 jobs completed" with blue checkmark, but below under "Annotations" all have "Process completed with exit code 1" and a failure symbol. Any ideas?

Yes. If I understand correctly, that's because the `git diff` rrt

**Comment by Rodrigo-Tenorio** [comment]
Accidentally closed this @dbkeitel

**Review by dbkeitel — DISMISSED** [review]
should be possible to clean this up, but I'll approve it for now -> for followup #472

File: .github/workflows/check_notebooks.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        $GITHUB_WORKSPACE/examples/tutorials/*ipynb
+    - name: Install package and development dependencies
+      if: steps.changed.outcome == 'failure'
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e ${GITHUB_WORKSPACE}[style,test,wheel]
+    - name: Make sure tutorials run
+      if: steps.changed.outcome == 'failure'
+      run: |
+        (cd .. && pytest --nbmake $GITHUB_WORKSPACE/examples/tutorials/*.ipynb)
```
Comment (Rodrigo-Tenorio) [inline .github/workflows/check_notebooks.yml:41]:
```suggestion
    - name: Install package and development dependencies
      run: |
        python -m pip install --upgrade pip
        pip install -e ${GITHUB_WORKSPACE}[style,test,wheel]
    - name: Make sure tutorials run
      run: |
        (cd .. && pytest --nbmake $GITHUB_WORKSPACE/examples/tutorials/*.ipynb)
```

Comment (Rodrigo-Tenorio) [reply .github/workflows/check_notebooks.yml:41]: Closes #472 as it was unnecessary in the end: Notebooks should always be tested.

**Pull request merged**